openshift/rh-che.app.yaml

```diff
@@ -359,6 +359,6 @@ objects:
       name: rhche-host
 parameters:
 - name: IMAGE
-  value: rhche/rhche-server
+  value: registry.devshift.net/che/rh-che-server
```

Comment:
this is centOS based, right?

Reply:
@riuvshin good point. I have double checked and it's centos:

```bash
$ docker run -ti --entrypoint=bash registry.devshift.net/che/rh-che-server
[root@8be18b5606a9 /]# cat /etc/centos-release
CentOS Linux release 7.5.1804 (Core)
```